com.gitlab.j0chn.nextcloud_password_client.json

```diff
@@ -0,0 +1,90 @@
+{
+    "app-id": "com.gitlab.j0chn.nextcloud_password_client",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "nextcloud_password_client",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
+        "--filesystem=xdg-download"
+    ],
+    "modules": [
+        {
+            "name": "libjsoncpp",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=release",
+                "-DBUILD_SHARED_LIBS=ON"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/open-source-parsers/jsoncpp/archive/refs/tags/1.7.5.tar.gz",
+                    "sha256": "4338c6cab8af8dee6cdfd54e6218bd0533785f552c6162bb083f8dd28bf8fbbe"
+                }
+            ]
+        },
+
+        {
+            "name": "libsodium",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.libsodium.org/libsodium/releases/libsodium-1.0.18.tar.gz",
+                    "sha256": "6f504490b342a4f8a4c4a02fc9b866cbef8622d5df4e5452b46be121e46636c1"
+                }
+            ]
+        },
+        {
+            "name": "libsecret",
+            "buildsystem": "meson",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=release",
+                "-DBUILD_SHARED_LIBS=ON"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/GNOME/libsecret/archive/refs/tags/0.20.4.tar.gz",
+                    "sha256": "ca34e69b210df221ae5da6692c2cb15ef169bb4daf42e204442f24fdb0520d4b"
+                }
+            ]
+        },
+        {
+            "name": "nextcloud_password_client",
+            "buildsystem": "simple",
+            "only-arches": [
+                "x86_64"
+            ],
+            "build-commands": [
+                "cp -r nextcloud_password_client /app/nextcloud_password_client",
+                "chmod +x /app/nextcloud_password_client/",
+                "install -Dm644 nextcloud_password_client/npc_logo.svg /app/share/icons/hicolor/scalable/apps/com.gitlab.j0chn.nextcloud_password_client.svg",
+                "mkdir -p /app/bin",
+                "ln -s /app/nextcloud_password_client/nextcloud_password_client /app/bin/nextcloud_password_client",
```

Comment:
this create a dangling link.

Have you ever tried to run the build?

Reply:
just to clarify the binary is NOT in /app/nextcloud_password_client/nextcloud_password_client

Reply:
```
flatpak run com.gitlab.j0chn.nextcloud_password_client
bwrap: execvp nextcloud_password_client: No such file or directory
```
I thought that would be a problem of local building.

Reply:
no. it should produce the exact same thing. If it doesn't work locally it's unlike to work on the buiders (the buiders have more check though)

Reply:
still not working because the symlink is still danging. The binary name from the tarball is `nextcloud_passwords_client`

**note** it's passwor**s** (plural)